docs/src/main/asciidoc/funqy-http.adoc

```diff
@@ -11,6 +11,13 @@ include::./attributes.adoc[]
 The guide walks through quickstart code to show you how you can deploy Funqy as a
 standalone service and invoke on Funqy functions using HTTP.
 
+WARNING: The Funqy HTTP binding is not a replacement for REST over HTTP.  Because Funqy
+needs to be portable cross a lot of different protocols and function providers its HTTP binding
+is very minimalistic and you will lose REST features like linking and the ability to leverage
+HTTP features like cache-control and conditional GETs.  You may want to consider using Quarkus's
+JAX-RS, Spring MVC, or Vert.x Web Reactive Route support instead, although Funqy will have less overhead
```

Comment:
Same comment about missing Servlet and not including Spring MVC

Reply:
Disagree.  No recommending servlet.  and we do support Spring MVC in quarkuks